### PR TITLE
chore(integrations); move from id to name

### PR DESCRIPTION
**Pull request by migmartri**
- Attach by workflow and integration name
- Filter attachments by workflow name

refs #1027

### CODE REVIEW

File: app/controlplane/pkg/data/orgmetrics.go [hunk 1]
```diff
@@ -43,10 +43,10 @@ func NewOrgMetricsRepo(data *Data, l log.Logger) biz.OrgMetricsRepo {
 }
 
 func (repo *OrgMetricsRepo) RunsTotal(ctx context.Context, orgID uuid.UUID, tw time.Duration) (int32, error) {
-	total, err := orgScopedQuery(repo.data.DB, orgID).
+	total, err := orgScopedQuery(repo.data.DB.Debug(), orgID).
 		QueryWorkflows().
 		QueryWorkflowruns().
-		Where(workflowrun.CreatedAtGTE(time.Now().Add(-tw))).
+		Where(workflowrun.CreatedAtIn(time.Now(), time.Now().Add(-tw))).
```
Comment (javirln) [inline app/controlplane/pkg/data/orgmetrics.go:49]: This is leftover right?

Comment (javirln) [inline app/controlplane/pkg/data/orgmetrics.go:49]: These are leftovers right?

**Review by javirln — CHANGES_REQUESTED** [review]
I think there are leftovers needed to be removed 👀

File: app/controlplane/pkg/data/orgmetrics.go (hunk 1 above)
Comment (migmartri) [reply app/controlplane/pkg/data/orgmetrics.go:49]: Oh man, sorry about that. Great catch! ;)

**Review by javirln — APPROVED** [review]

**Pull request merged**